From 7b3e37b692c9283914f4a36b993b2ca63e21622c Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Tue, 21 Feb 2017 09:53:46 +0100
Subject: [PATCH 14/15] configs: score: Add configuration for pca953x port
 expander

Add information for the pca953x port expander. This
information contains i2c slave address, init direction
mask and the initial values for the output pins.

Ticket #1700
---
 include/configs/score.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/include/configs/score.h b/include/configs/score.h
index 58fb414..70302e3 100644
--- a/include/configs/score.h
+++ b/include/configs/score.h
@@ -111,6 +111,15 @@
 #define CONFIG_PMIC_I2C_SLAVE		0x8
 #endif
 
+#define CONFIG_PCA953X
+#define CONFIG_CMD_PCA953X
+#define CONFIG_PCA953X_BUS_NUM 2
+#define CONFIG_PCA953X_OUT_MASK 0xecdf
+#define CONFIG_PCA953X_OUT_INIT 0x8001
+#define CONFIG_CMD_PCA953X_INFO
+#define CONFIG_SYS_I2C_PCA953X_ADDR 0x20
+#define CONFIG_SYS_I2C_PCA953X_WIDTH { {0x20, 16} }
+
 /* Command definition */
 #include <config_cmd_default.h>
 
-- 
2.1.4

